### ISSUE TITLE
Jars listed in Module XML are not de-duplicated by scope 

### ISSUE THREAD

**Issue by benmusson** [inductiveautomation/ignition-module-tools#52]
Currently, the plugin allows for generation of `module.xml` files that list the same jar multiple times.
I would expect the plugin to combine these entries together by combining the scopes.

---

Current behavior, notice how `kotlin-stdlib-2.0.0.jar` has entries for `C`, `CDG`, `D`, and `G`.
```xml
<jar scope="C">core-client-0.1.0.jar</jar>
<jar scope="C">kotlin-stdlib-2.0.0.jar</jar>
<jar scope="C">annotations-13.0.jar</jar>
<jar scope="C">if97-steam-tables-client-0.1.0.jar</jar>
<jar scope="CDG">core-common-0.3.0.jar</jar>
<jar scope="CDG">if97-2.0.0.jar</jar>
<jar scope="CDG">kotlin-stdlib-2.0.0.jar</jar> 
<jar scope="CDG">annotations-13.0.jar</jar>
<jar scope="CDG">if97-steam-tables-common-0.1.0.jar</jar>
<jar scope="D">core-designer-0.3.0.jar</jar>
<jar scope="D">kotlin-stdlib-2.0.0.jar</jar>
<jar scope="D">annotations-13.0.jar</jar>
<jar scope="D">if97-steam-tables-designer-0.1.0.jar</jar>
<jar scope="G">core-gateway-0.3.0.jar</jar>
<jar scope="G">kotlin-stdlib-2.0.0.jar</jar>
<jar scope="G">annotations-13.0.jar</jar>
<jar scope="G">if97-steam-tables-gateway-0.1.0.jar</jar>
```

---

Expected behavior, `kotlin-stdlib-2.0.0.jar` should have a single entry for `CDG` (the maximum applied extent of the scope).
```xml
<jar scope="C">core-client-0.1.0.jar</jar>
<jar scope="C">if97-steam-tables-client-0.1.0.jar</jar>
<jar scope="CDG">annotations-13.0.jar</jar>
<jar scope="CDG">core-common-0.3.0.jar</jar>
<jar scope="CDG">if97-2.0.0.jar</jar>
<jar scope="CDG">if97-steam-tables-common-0.1.0.jar</jar>
<jar scope="CDG">kotlin-stdlib-2.0.0.jar</jar>
<jar scope="D">core-designer-0.3.0.jar</jar>
<jar scope="D">if97-steam-tables-designer-0.1.0.jar</jar>
<jar scope="G">core-gateway-0.3.0.jar</jar>
<jar scope="G">if97-steam-tables-gateway-0.1.0.jar</jar>
```

**Comment by brianeray** [comment]
For internal tracking: IGN-10612.

**Comment by brianeray** [comment]
Fixed by #53.
